### DISCUSSION TITLE
Master graduation project question: Adaptive Core Streamlines

### DISCUSSION THREAD

**Discussion by paulzelina** [dipy/dipy#3131] [3. Community Lounge]
Hello everyone, 
My name is Paul and I am currently undergoing my Master's thesis. The goal of my project is to expand on current core streamline extraction methods, and then obviously evaluate the new methods. In order to compare the FA along different core streamlines, I am trying to use dsa.afq_profile with the fa as a param. However, it seems that the output is exactly the same for different core streamlines and orientations, which leads me to believe I am doing something wrong. Is there anyone willing to help/guide me a bit with these methods/formats? Thank you in advance! 
Kind regards,
Paul Zelina

**Comment by arokem** [comment]
Hi Paul, 

Thanks for posting. Would you like to share the code that you are working with? It sounds like you think there is something wrong with the code, but it's hard to judge in the abstract.

**Reply by paulzelina** [reply]
Hello,
I am sorry for the late response. Is there a possibility of sending the code by email?
Kind regards,
Paul Zelina
